python/tests/visualization/test_draw.py

```diff
@@ -24,13 +24,9 @@ def test_draw():
 
     @cudaq.kernel
     def bar(qvec: cudaq.qview):
-        # FIXME https://github.com/NVIDIA/cuda-quantum/issues/1734
-        # rx(np.e, qvec[0])
-        rx(2.71828182845904523536028, qvec[0])
+        rx(np.e, qvec[0])
```

Comment:
Can we add a test for `euler_gamma` as well? and maybe a negative test for something that is not supported?

Reply:
Addressed in 96d4ad8310c72779d7a22a262c8c24f026d21c46